src/utils/components/DescriptionItem/DescriptionItemHeader.tsx

```diff
@@ -15,6 +15,7 @@ type DescriptionItemHeaderProps = {
   bodyContent: ReactNode;
   breadcrumb?: string;
   descriptionHeader: string;
+  hasIcon?: boolean;
```

Comment:
Hi :) 
nit suggestion: instead of passing hasIcon as boolean, add it as a ReactNode with null default value.
It's hard to guess that `bodyContent` is also meant to be an Icon.

Reply:
The word `content` refers to text, I'd go with just `icon`, that should be clear enough :)

Reply:
Well, I am not sure. It provides icon together with popover text, so using just `icon` doesn't feel to me as good enough. And iconWithPopover feels to me unnecessary long :D